docs/Makefile

```diff
@@ -7,7 +7,7 @@
 # (except for '--keep-going', failing fast is prefered on local builds)
 # We also remove '-E' to reduce the time of rebuilding reference indexes
 # on each build.
-SPHINXOPTS    = -T -j auto -W
+SPHINXOPTS    = -T -W
```

Comment:
Pretty sure that `-j auto` is the default in our Sphinx builder?

Reply:
Oh yea, I was getting errors on this locally, but I just needed to update my depednencies.